dll/win32/shell32/stubs.cpp

```diff
@@ -136,6 +145,19 @@ WINAPI
 SHMultiFileProperties(IDataObject *pDataObject, DWORD dwFlags)
 {
     FIXME("SHMultiFileProperties() stub\n");
+
+    // Temporary workaround to display a property sheet if possible
+    if (DataObject_GetHIDACount(pDataObject) == 1)
+        return SHELL32_ShowPropertiesDialog(pDataObject);
+
+    if (pDataObject)
+    {
+        HWND hWnd;
+        if (FAILED(IUnknown_GetWindow(pDataObject, &hWnd))) // Will probably not work but we have no other option
+            hWnd = NULL;
+        SHELL_ErrorBox(hWnd, HRESULT_FROM_WIN32(ERROR_NOT_SUPPORTED));
```

Comment:
Does Windows also pop up a message box in that failure case?

Reply:
Windows does not have that failure case.

If I pass something that is not really a valid file (the desktop or a file that does not exist, it still displays the propsheet dialog:

![screenshot](https://github.com/reactos/reactos/assets/138629473/02766551-eff0-42a9-b8ca-ea54490b2806)

If I pass a NULL dataobject, it does nothing. If I pass a dataobject with zero items, Windows displays a MessagBox that it cannot display the properties for this item.